webpack.config.js

```diff
@@ -32,6 +36,7 @@ const BABEL_PRESET_ENV_CONFIG = {
     useBuiltIns: 'usage',
     modules: false,
 };
+const PRELOAD_ASSETS = ['billing', 'shipping', 'payment'];
```

Comment:
What does this do? Does it mean we now load assets before we actually get on the step?

Reply:
@animesh1987 We used to preload all assets listed in the `dynamicChunks` section of `manifest.json`. However, as `manifest.json` now includes the dynamic chunks for `loader.js`, which contains translation JSON files in various languages (e.g., `translations-es-1.525.0.js`), we need to limit preloading to only the necessary ones (`billing`, `shipping` and `payment`).